src/bin/cybergrape/main.rs

```diff
@@ -26,6 +27,8 @@ use std::{
     time::Duration,
 };
 
+const BAUD_RATE: u32 = 115200;
+
 // Example:
 // cargo run --bin cybergrape --
 //                            --samp    44100
```

Comment:
Should we change this to reflect the interface described with the `clap` arguments?